python/hsfs/core/job.py

```diff
@@ -58,3 +59,6 @@ def executions(self):
     @property
     def href(self):
         return self._href
+
+    def run(self, write_options=None):
```

Comment:
We shouldn't leak "feature store concepts" like `write_options` to the jobs api.